mssqlcli/mssql_cli.py

```diff
@@ -269,19 +266,12 @@ def set_main_mssqlcli_client(self, mssqlcli_client):
         self.mssqlcliclient_main = mssqlcli_client
 
     def connect_to_database(self):
-
-        try:
-            owner_uri, error_messages = self.mssqlcliclient_main.connect_to_database()
-            if not owner_uri and error_messages:
-                click.secho('\n'.join(error_messages),
-                            err=True,
-                            fg='yellow')
-                sys.exit(1)
-
-        except Exception as e:
```

Comment:
Where is this exception handling logic gone? What happens if an unexpected exception occurs now?

Reply:
Exception handling is gone because it is already handled from upstream calls. If an exception occurs, it provides a value to the `error_messages` variable in this function.

PEP 8 also discourages using generic exception handling like what existed here. @kevcunnane what are your thoughts?